web_design.css

```diff
@@ -2435,31 +2435,71 @@ hr
 
 /* CSS3 on reddit! */
 
+@media (max-width: 800px){
+    #header-bottom-right{
+        top: 176px;
+        width: 100%;
+        text-align: center;
+        margin-right: 0;
+        padding-bottom: 15px;
+        padding-right: 0;
+    }
+
+    #header-bottom-right:hover
+    {
+        width: 100%;
+    }
+
+    .content{
+        margin-top: 65px;
+    }
+
+    .seachpane, .searchfacets, .menuarea{
+        margin-right: 10px;
+    }
+}
+
 @media (max-width: 800px) and (min-width: 601px)
 {
-	.link, .commentarea { margin-right: 10px; }
-	.side { display: none; }
+    .link, .commentarea { margin-right: 10px; }
```

Comment:
not a fan of including whitespace changes, try to avoid that in the future. Check out the [sourcetree](http://www.sourcetreeapp.com/) app if you're having a hard time in terminal.